### DISCUSSION TITLE
Error launching codespace

### DISCUSSION THREAD

**Discussion by mah51** [community/community#18105] [Codespaces] [answered]
I have deleted and recreated multiple times and on start it will come up with this error:

Error details:
VSCodeWorkbench mount failed with "script "https://main.vscode-cdn.net/stable/c3511e6c69bb39013c4a4b7b9566ec1ca73fc4d5/out/vs/loader.j
s" failed to load."

refreshing the page doesn't seem to do anything.

**Comment by masonbesmer** [comment, marked as answer]
This is server side on github apparently, a friend and I am also experiencing this issue. I guess this is what I get for depending on the cloud for devving haha.

**Reply by mah51** [reply]
Oh that is bad timing! I just got the email that I am on the beta and this is the first container I set up. Thought it was a me thing... I guess we shall sit tight :).

**Reply by masonbesmer** [reply]
Should be good now.

**Reply by mah51** [reply]
it does! thank you
